protobuf/concept.proto

```diff
@@ -22,212 +22,101 @@ option java_outer_classname = "ConceptProto";
-
-                // Relation iterator responses
-                Relation.GetPlayers.Iter.Req relation_getPlayers_iter_req = 200;
-                Relation.GetPlayersByRoleType.Iter.Req relation_getPlayersByRoleType_iter_req = 201;
-
-                // Attribute iterator requests
-                Attribute.GetOwners.Iter.Req attribute_getOwners_iter_req = 302;
-            }
         }
         message Res {
             oneof res {
-                // Thing iterator responses
-                Thing.GetHas.Iter.Res thing_getHas_iter_res = 100;
-                Thing.GetRelations.Iter.Res thing_getRelations_iter_res = 101;
-                Thing.GetPlays.Iter.Res thing_getPlays_iter_res = 102;
-
-                // Relation iterator responses
-                Relation.GetPlayers.Iter.Res relation_getPlayers_iter_res = 200;
-                Relation.GetPlayersByRoleType.Iter.Res relation_getPlayersByRoleType_iter_res = 201;
-
-                // Attribute iterator responses
-                Attribute.GetOwners.Iter.Res attribute_getOwners_iter_res = 300;
+                Thing thing = 1;
             }
         }
     }
-}
 
-message TypeMethod {
-    message Req {
-        string label = 1;
-        string scope = 2;
-        oneof req {
-            // Type method requests
-            Type.Delete.Req type_delete_req = 100;
-            Type.SetLabel.Req type_setLabel_req = 101;
-            Type.IsAbstract.Req type_isAbstract_req = 102;
-            Type.GetSupertype.Req type_getSupertype_req = 103;
-            Type.SetSupertype.Req type_setSupertype_req = 104;
-
-            // RoleType method requests
-            RoleType.GetRelation.Req roleType_getRelation_req = 200;
-
-            // ThingType method requests
-            ThingType.SetAbstract.Req thingType_setAbstract_req = 300;
-            ThingType.UnsetAbstract.Req thingType_unsetAbstract_req = 301;
-            ThingType.SetOwns.Req thingType_setOwns_req = 302;
-            ThingType.UnsetOwns.Req thingType_unsetOwns_req = 303;
-            ThingType.SetPlays.Req thingType_setPlays_req = 304;
-            ThingType.UnsetPlays.Req thingType_unsetPlays_req = 305;
-
-            // EntityType method requests
-            EntityType.Create.Req entityType_create_req = 400;
-
-            // RelationType method requests
-            RelationType.Create.Req relationType_create_req = 500;
-            RelationType.GetRelatesForRoleLabel.Req relationType_getRelatesForRoleLabel_req = 501;
-            RelationType.SetRelates.Req relationType_setRelates_req = 502;
-            RelationType.UnsetRelates.Req relationType_unsetRelates_req = 503;
-
-            // AttributeType method requests
-            AttributeType.Put.Req attributeType_put_req = 600;
-            AttributeType.Get.Req attributeType_get_req = 601;
-            AttributeType.GetRegex.Req attributeType_getRegex_req = 602;
-            AttributeType.SetRegex.Req attributeType_setRegex_req = 603;
```

Comment:
Given that these TypeMethods are just messages that take in a Type, and there was already a message called Type, we decided to move them there for simplicity.